tensorflow/python/client/session_test.py

```diff
@@ -1742,6 +1742,12 @@ def testAddFunctionToGrpcSession(self):
     server = server_lib.Server.create_local_server()
     self.runTestAddFunctionToSession(server.target)
 
+  def testAutoConvertAndCheckData(self):
+    with self.test_session() as sess:
+      a = array_ops.placeholder(tf.string)
+      with self.assertRaises(TypeError) as e:
```

Comment:
Style nit: can you please change this line to the following and remove line 1750?

```python
with self.assertRaisesRegexp(TypeError, 'Type of feed value 1 with type in is not'):
```